variables.tf

```diff
@@ -59,9 +59,11 @@ variable "vcs_repo" {
   default     = null
 
   type = object({
-    identifier     = string
-    oauth_token_id = string
-    branch         = optional(string)
+    branch                     = optional(string)
+    github_app_installation_id = optional(string)
+    identifier                 = string
+    ingress_submodules         = optional(bool)
+    oauth_token_id             = optional(string)
```

Comment:
potentially missing the `vcs_repo_enable` var here?

Reply:
can this be `true` by default?

Reply:
`vcs_repo_enable` is a local variable so it can't be there
Regarding `true` by default, I think it can be enabled in the `vcs_repo` conditional block but I need to rewrite and verify that (it was the most tricky part)